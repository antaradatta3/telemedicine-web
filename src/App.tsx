import React from 'react';
import { akitaDevtools } from '@datorama/akita';

import DocumentTitle from './components/Common/DocumentTitle';
import Search from './components/Search/index';
import Home from './components/Home/index';
import './App.css';

import {
  BrowserRouter as Router,
  Switch,
  Route,
} from 'react-router-dom';

akitaDevtools();

const App = () => {
  return (
    <div className="App">
      <Router>
        <Switch>
          <Route exact path="/">
            <Home />
          </Route>
          <Route path="/dashboard">
            <Search />
          </Route>
        </Switch>
      </Router>
    </div>
  );
};

export default App;
