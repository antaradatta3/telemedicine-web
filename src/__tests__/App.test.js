import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';

import App from '../App';

describe('Component: App', () => {
  const props = {};

  it('App renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App {...props} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('App matches snapshot', () => {
    const component = mount(<App {...props} />);
    expect(component).toMatchSnapshot();
  });
});
