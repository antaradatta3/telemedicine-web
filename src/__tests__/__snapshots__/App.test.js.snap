// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App App matches snapshot 1`] = `
<App>
  <div
    className="App"
  >
    <header
      className="App-header"
    >
      <DocumentTitle
        listen={true}
        prefix="Reddit Threads"
      >
        <h3>
          Reddit Threads - 0, 0
        </h3>
      </DocumentTitle>
      <div>
        <Search>
          <div>
            <SearchForm
              search={[Function]}
            >
              <div
                className="form-inline"
              >
                <input
                  className="form-control"
                  onChange={[Function]}
                  placeholder="Search Thread"
                  type="text"
                  value=""
                />
                <button
                  className="btn btn-primary"
                  onClick={[Function]}
                  type="button"
                >
                  Search
                </button>
              </div>
            </SearchForm>
          </div>
        </Search>
      </div>
    </header>
    <Home>
      <div>
        <div>
          <ThreadList>
            <div>
              <div />
            </div>
          </ThreadList>
        </div>
        <div>
          <Thread>
            <div />
          </Thread>
        </div>
      </div>
    </Home>
  </div>
</App>
`;
